channels-rke2.yaml

```diff
@@ -97,10 +97,10 @@ releases:
         version: v3.13.300-build2021022306
       rke2-calico:
         repo: rancher-rke2-charts
-        verion: v3.19.1-105
+        verion: v3.19.2-201
```

Comment:
Oh wait, why does this say verion instead of version. Is that a bug?

Reply:
good catch! I did not see that. Let me change it too

Reply:
Seems like it was introduced in https://github.com/rancher/kontainer-driver-metadata/commit/9b7f59b814887e316e67dfb0edbe334db79bbbc8.

I'm not sure if there are any other repercussions for this, so maybe we should double check with the rest of the team. Could be related to an open issue?

Reply:
I'd expect that calico does not deploy correctly but I haven't tested deploying from Rancher. You tested it when you created this issue https://github.com/rancher/rke2/issues/1541, right?